src/View/ViewableData.php

```diff
@@ -546,8 +553,11 @@ protected function objCacheClear()
      */
     public function obj($fieldName, $arguments = [], $cache = false, $cacheName = null)
     {
+        if ($cacheName !== null) {
+            Deprecation::notice('5.4.0', 'The $cacheName parameter has been deprecated and will be removed');
+        }
```

Comment:
That param was never used, and it's easier to reason about the cache if the cache name is always derived from the field name and arguments.